code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -1,275 +1,9 @@
-	maxHealth = 170
-	health = 170
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 0
-	speed = 1
-
-/mob/living/simple_animal/hostile/syndicate/ranged/space/Initialize(mapload)
-	. = ..()
-	ADD_TRAIT(src, TRAIT_SPACEWALK, INNATE_TRAIT)
-	set_light(4)
-
-/mob/living/simple_animal/hostile/syndicate/ranged/space/stormtrooper
-	icon_state = "syndicate_stormtrooper_pistol"
-	icon_living = "syndicate_stormtrooper_pistol"
-	name = "Syndicate Stormtrooper"
-	maxHealth = 250
-	health = 250
-
-/mob/living/simple_animal/hostile/syndicate/ranged/smg
-	rapid = 2
-	icon_state = "syndicate_smg"
-	icon_living = "syndicate_smg"
-	casingtype = /obj/item/ammo_casing/c45
-	projectilesound = 'sound/weapons/gun/smg/shot.ogg'
-
-/mob/living/simple_animal/hostile/syndicate/ranged/smg/pilot //caravan ambush ruin
-	name = "Syndicate Salvage Pilot"
-	loot = list(/obj/effect/mob_spawn/corpse/human/syndicatesoldier)
-
-/mob/living/simple_animal/hostile/syndicate/ranged/smg/space
-	icon_state = "syndicate_space_smg"
-	icon_living = "syndicate_space_smg"
-	name = "Syndicate Commando"
-	maxHealth = 170
-	health = 170
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 0
-	speed = 1
-
-/mob/living/simple_animal/hostile/syndicate/ranged/smg/space/Initialize(mapload)
-	. = ..()
-	ADD_TRAIT(src, TRAIT_SPACEWALK, INNATE_TRAIT)
-	set_light(4)
-
-/mob/living/simple_animal/hostile/syndicate/ranged/smg/space/stormtrooper
-	icon_state = "syndicate_stormtrooper_smg"
-	icon_living = "syndicate_stormtrooper_smg"
-	name = "Syndicate Stormtrooper"
-	maxHealth = 250
-	health = 250
-
-/mob/living/simple_animal/hostile/syndicate/ranged/shotgun
-	rapid = 2
-	rapid_fire_delay = 6
-	minimum_distance = 3
-	icon_state = "syndicate_shotgun"
-	icon_living = "syndicate_shotgun"
-	casingtype = /obj/item/ammo_casing/shotgun/buckshot //buckshot (up to 72.5 brute) fired in a two-round burst
-
-/mob/living/simple_animal/hostile/syndicate/ranged/shotgun/space
-	icon_state = "syndicate_space_shotgun"
-	icon_living = "syndicate_space_shotgun"
-	name = "Syndicate Commando"
-	maxHealth = 170
-	health = 170
-	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_plas" = 0, "max_plas" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
-	minbodytemp = 0
-	speed = 1
-
-/mob/living/simple_animal/hostile/syndicate/ranged/shotgun/space/Initialize(mapload)
-	. = ..()
-	ADD_TRAIT(src, TRAIT_SPACEWALK, INNATE_TRAIT)
-	set_light(4)
-
-/mob/living/simple_animal/hostile/syndicate/ranged/shotgun/space/stormtrooper
-	icon_state = "syndicate_stormtrooper_shotgun"
-	icon_living = "syndicate_stormtrooper_shotgun"
-	name = "Syndicate Stormtrooper"
-	maxHealth = 250
-	health = 250
-
 ///////////////Misc////////////
 
 /mob/living/simple_animal/hostile/syndicate/civilian
```

Comment:
this won't have its icon anymore no? because you deleted its parent

Reply:
i probably should remove it instead because its completely unused